packages/documentation/plugins/mdx-pages/gatsby-node.js

```diff
@@ -66,6 +68,9 @@ exports.createPages = async ({ graphql, actions }) => {
         context: {
           id: node.id,
           source: 'component',
+          frontmatter: {
+            ...node.frontmatter,
```

Comment:
should only need to add new front matter to the query on the MDX components with this.